docs/tutorials/09_properties.ipynb

```diff
@@ -0,0 +1,219 @@
+   "execution_count": 2,
+   "id": "fb1d2956",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AngularMomentum is in problem.properties: False\n",
+      "Adding AngularMomentum to problem.properties...\n",
+      "AngularMomentum is in problem.properties: True\n",
+      "Discarding AngularMomentum from problem.properties...\n",
+      "AngularMomentum is in problem.properties: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.properties import AngularMomentum\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)\n",
+    "\n",
+    "print(\"Adding AngularMomentum to problem.properties...\")\n",
+    "base_problem.properties.add(AngularMomentum(2))\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)\n",
+    "\n",
+    "print(\"Discarding AngularMomentum from problem.properties...\")\n",
+    "base_problem.properties.discard(AngularMomentum)\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b72239b",
+   "metadata": {},
+   "source": [
+    "The specific subclasses of `BaseProblem` for the various stacks such as [electronic structure](01_electronic_structure.ipynb), [vibrational structure](02_vibrational_structure.ipynb) or [lattice models](10_lattice_models.ipynb) make things even simpler for you, because they provide direct attributes for the built-in properties of Qiskit Nature:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6c251832",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.properties.particle_number.ParticleNumber object at 0x14bfa1aa5b80>\n",
+      "<qiskit_nature.second_q.properties.angular_momentum.AngularMomentum object at 0x14bfa1aa5a60>\n",
+      "<qiskit_nature.second_q.properties.magnetization.Magnetization object at 0x14bfa1aa5ac0>\n",
+      "<qiskit_nature.second_q.properties.dipole_moment.ElectronicDipoleMoment object at 0x14bfa1aa5f70>\n",
+      "None\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "\n",
+    "es_problem = PySCFDriver().run()\n",
+    "\n",
+    "print(es_problem.properties.particle_number)\n",
+    "print(es_problem.properties.angular_momentum)\n",
+    "print(es_problem.properties.magnetization)\n",
+    "print(es_problem.properties.electronic_dipole_moment)\n",
+    "print(es_problem.properties.electronic_density)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dca06da",
+   "metadata": {},
+   "source": [
+    "You can see, that all properties come pre-populated except for the `ElectronicDensity`. But this is easy to add:"
```

Comment:
If user adds such a property what should be their expectation of what happens when the problem is run. These all end up producing auxillary operators whose observable is measured and returned in the result?

Reply:
Yes exactly :+1: 

But the user will need to extract the evaluated auxiliary operators manually since the result classes do not expose arbitrary data fields (and unless the user also implement `.interpret()` they would not be populated anyways